src/components/ModalBox/ModalBox.scss

```diff
@@ -0,0 +1,67 @@
+$primary: #2D6686;
+$default:#FFF;
+$secondaryColor: #F34579;
```

Comment:
Please use the colors used in the MainStyle.scss file
Simply import the MainStyle.scss file and the variables will be available to you.

`@import 'assets/MainStyle.scss'`